internal/scripts/bin/hackenv_fixlabels

```diff
@@ -8,6 +8,17 @@ datahome="$datahome/hackenv"
 
 shared="$(realpath "$datahome/shared")"
 
+printf "Trying to fix SElinux labels...\n" >&2
+
+if ! command -v semanage &> /dev/null; then
+	printf "SElinux coreutils binaries could not be found.\n" >&2
+	if command -v yum &> /dev/null; then
+		sudo yum -y install policycoreutils-python-utils
+	else
+		printf "Exiting.\n" >&2 && exit 0
```

Comment:
I would personally refrain from installing packages in our fix command. Maybe we can suggest installing the correct package instead?

Reply:
Ok, I print the suggestion instead now.